tests/executors/test_celery_kubernetes_executor.py

```diff
@@ -58,182 +64,104 @@ def test_start(self):
         celery_executor_mock.start.assert_called()
         k8s_executor_mock.start.assert_called()
 
-    def test_queue_command(self):
-        command = ['airflow', 'run', 'dag']
-        priority = 1
-        queue = 'default'
-
-        def when_using_k8s_executor():
-            celery_executor_mock = mock.MagicMock()
-            k8s_executor_mock = mock.MagicMock()
-            cke = CeleryKubernetesExecutor(celery_executor_mock, k8s_executor_mock)
-
-            simple_task_instance = mock.MagicMock()
-            simple_task_instance.queue = CeleryKubernetesExecutor.KUBERNETES_QUEUE
-
-            cke.queue_command(simple_task_instance, command, priority, queue)
-
-            k8s_executor_mock.queue_command.assert_called_once_with(
-                simple_task_instance, command, priority, queue
-            )
-            celery_executor_mock.queue_command.assert_not_called()
-
-        def when_using_celery_executor():
-            celery_executor_mock = mock.MagicMock()
-            k8s_executor_mock = mock.MagicMock()
-            cke = CeleryKubernetesExecutor(celery_executor_mock, k8s_executor_mock)
-
-            simple_task_instance = mock.MagicMock()
-            simple_task_instance.queue = 'non-kubernetes-queue'
+    @parameterized.expand(
+        [
+            ('other-queue',),
```

Comment:
nit: let's use the original queue name "'non-kubernetes-queue"? to be consistent with other related test cases, also being more explicit about what's tested here.

Reply:
how about `'any-queue'` or `'any-other-queue'` -- because the behavior were testing is the difference in handling between _the_ kubernetes queue (there is only one) and any queue besides the kubernetes queue

or `'arbitrarily-named-queue'` or `'could-be-anything'`:) 

my only hesitation with `'non-kubernetes-queue'` is that there's some possibility it could be interpreted to mean that there is a particular designated queue, namely `'non-kubernetes-queue'` which routes to celery, and otherwise it goes to kubernetes queue

my making the naming more generic, or more obviously arbitrary, i think it adds to readability.

let me know what you think.  happy to defer to your judgment on this.

Reply:
i went with `'any-other-queue'`